src/main/scala/io/iohk/ethereum/jsonrpc/server/http/JsonRpcHttpServer.scala

```diff
@@ -79,11 +80,15 @@ trait JsonRpcHttpServer extends Json4sSupport {
   }
 
   private def handleRequest(request: JsonRpcRequest) = {
-    complete(jsonRpcController.handleRequest(request))
+    complete(jsonRpcController.handleRequest(request).runToFuture(MonixGlobal))
```

Comment:
Why not https://monix.io/api/3.2/monix/execution/Scheduler$.html#global:monix.execution.Scheduler then? That one is not available in implicit scope

Reply:
Yeach previously I needed to rename globacl Scheduler from monix to avoid name class. But since c02157d1939e162ec92d38b174be10a73e2046ef there is no name clash. Fixed: https://github.com/input-output-hk/mantis/pull/756/commits/490f30a57975707055b5df49d3bbad05e015ee4b